experimental/docker-stacks-and-bundles.md

```diff
@@ -52,7 +52,7 @@ Options:
 Let's deploy the stack created before:
 
 ```bash
-# docker deploy vossibility-stack
+# docker deploy --bundle-file vossibilitystack.dab vossibility
```

Comment:
The `--bundle-file` doesn't match the output below, which says:

```
Loading bundle from vossibility-stack.dab
```

Also, the _name_ given (see the output below) is `vossibility-stack`, not `vossibility`


So this should be:

```bash
$ docker deploy --bundle-file vossibility-stack.dab vossibility-stack
```